recipes/blas_openblas/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: blas_openblas
+  version: 1.0.0
+
+build:
+  number: 0
+  track_features:
+    - blas_openblas
+
+test:
+  commands:
+    - conda list -n _test blas_openblas
```

Comment:
Setting the environment here feels like a horrific hack and quite fragile. Unfortunately, not setting it runs into this issue ( https://github.com/conda/conda-build/issues/910 ). So, not really sure there is another option ATM.